frontend/src/app/pages/exercises/exercise/shared/exercise-settings/exercise-settings-modal/exercise-settings-modal.component.html

```diff
@@ -62,24 +62,49 @@ <h4 class="modal-title">Einstellungen</h4>
     <div *ngIf="configuration$ | async as configuration" class="card">
         <div class="card-header">Übungsmodus</div>
         <div class="card-body">
-            Mit Vorsichtung üben?
-            <div class="form-switch">
-                <input
-                    [ngModel]="configuration.pretriageEnabled"
-                    (ngModelChange)="setPretriageFlag($event)"
-                    class="form-check-input"
-                    type="checkbox"
-                    data-cy="settingsPretriageCheckbox"
-                />
+            <div class="form-group mb-3">
+                Mit Vorsichtung üben?
+                <div class="form-switch">
+                    <input
+                        [ngModel]="configuration.pretriageEnabled"
+                        (ngModelChange)="setPretriageFlag($event)"
+                        class="form-check-input"
+                        type="checkbox"
+                        data-cy="settingsPretriageCheckbox"
+                    />
+                </div>
+            </div>
+
+            <div class="form-group mb-3">
+                Mit SK IV Patienten üben?
+                <div class="form-switch">
+                    <input
+                        [ngModel]="configuration.bluePatientsEnabled"
+                        (ngModelChange)="setBluePatientsFlag($event)"
+                        class="form-check-input"
+                        type="checkbox"
+                        data-cy="settingsBluePatientsCheckbox"
+                    />
+                </div>
             </div>
-            Mit SK IV Patienten üben?
-            <div class="form-switch">
+
+            <div class="form-group">
+                <label class="form-label mb-0">Präfix der Patienten-IDs:</label>
+                <span class="text-muted d-block">
+                    Mit den aktuellen Einstellungen haben Patienten-IDs die
+                    folgende Form:
+                    <code>{{ configuration.patientIdentifierPrefix }}1234</code>
+                </span>
                 <input
-                    [ngModel]="configuration.bluePatientsEnabled"
-                    (ngModelChange)="setBluePatientsFlag($event)"
-                    class="form-check-input"
-                    type="checkbox"
-                    data-cy="settingsBluePatientsCheckbox"
+                    #patientIdentifierPrefixInput="ngModel"
+                    [ngModel]="configuration.patientIdentifierPrefix"
+                    (appSaveOnTyping)="updatePatientIdentifierPrefix($event)"
+                    class="form-control"
+                    type="text"
+                    placeholder="B-"
```

Comment:
The Placeholder could be confusing because it could seem like the Prefix is B- when it is in fact not existing. Maybe put something like Your prefix here, e.g. B.

Reply:
Changes it in https://github.com/hpi-sam/digital-fuesim-manv/pull/1028/commits/ab547349eb22da6adf4e81c7a48f07aeccf0994c